core/include/tangram/tangram.h

```diff
@@ -61,16 +61,32 @@ struct SceneUpdate {
     std::string path;
     std::string value;
     SceneUpdate(std::string p, std::string v) : path(p), value(v) {}
+    SceneUpdate() {}
 };
 
+enum Error {
+    scene_update_path_not_found,
+    scene_update_path_yaml_syntax_error,
+    scene_update_value_yaml_syntax_error,
+};
+
+struct SceneUpdateError {
+    SceneUpdate update;
+    Error error;
+};
+
+using SceneUpdateErrorCallback = std::function<void(const SceneUpdateError&)>;
```

Comment:
No I am not too strong on it. Works for me.

Also just confirmed, we are using `Cb` in our internal code, but all public facing code is using `Callback`. So we are good here.
Thanks